src/libAtomVM/nifs.c

```diff
@@ -2544,9 +2544,30 @@ static term nif_erlang_list_to_binary_1(Context *ctx, int argc, term argv[])
     return bin_res;
 }
 
-static term nif_erlang_list_to_integer_1(Context *ctx, int argc, term argv[])
+static avm_int_t to_digit_index(avm_int_t character)
```

Comment:
Wasn't sure if helper functions belong here.

Reply:
that's the right place!